src/hiero_sdk_python/contract/contract_id.py

```
@@ -214,11 +212,10 @@ def to_string_with_checksum(self, client: "Client") -> str:
                 as checksums cannot be applied to EVM addresses.
         """
         if self.evm_address is not None:
-            raise ValueError("to_string_with_checksum cannot be applied to ContractId with evm_address")
+            raise ValueError(
+                "to_string_with_checksum cannot be applied to ContractId with evm_address"
+            )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)